akka-actor/src/main/scala/akka/io/InetAddressDnsResolver.scala

```diff
@@ -72,12 +73,12 @@ class InetAddressDnsResolver(cache: SimpleDnsCache, config: Config) extends Acto
         case None ⇒
           try {
             val answer = Dns.Resolved(name, InetAddress.getAllByName(name))
-            if (positiveTtl != Never) cache.put(answer, positiveTtl)
+            if (positiveTtl != Never) cache.put(answer, Duration.create(positiveTtl, TimeUnit.MILLISECONDS))
```

Comment:
```
import scala.concurrent.duration._

positiveTtl.millis
```

Reply:
better is to do it once, in `getTtl` so that `val positiveTtl: FiniteDuration`
same with negativeTtl